deploy/eck-integrations/values.yaml

```diff
@@ -0,0 +1,93 @@
+
+
+elasticsearch:
+  host: ""
+  user: ""
+  pass: ""
```

Comment:
Need support for API keys here

Reply:
Bitnami are a good point of reference for the above https://github.com/bitnami/charts/blob/main/bitnami/elasticsearch/values.yaml

Reply:
Yeah we'll definitely do that type of clean up - right now we're focused on a functional POC